kyo-data/shared/src/main/scala/kyo/Chunk.scala

```diff
@@ -30,7 +30,9 @@ sealed abstract class Chunk[A] extends Seq[A] derives CanEqual:
       * @return
       *   true if the Chunk contains no elements, false otherwise
       */
-    final override def isEmpty: Boolean = size == 0
+    final override def isEmpty: Boolean = length == 0
+
+    override def length: Int
```

Comment:
I noticed interface dispatches (itable stub) to obtain the size of a chunk. This PR introduces this class method so all internal uses can dispatch directly to it and avoid the indirection in `Seq`.